lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -172,8 +172,13 @@ def main():
         if set(dl1_params_input).intersection(disp_params):
             parameters_to_update.extend(disp_params)
 
+        if increase_nsb or increase_psf:
```

Comment:
nitpick, but the psf does not actually use the RNG, right?

Reply:
Why should it? It is a completely deterministic spread of light :D

Reply:
I think it is ok like that... despite the low number of photons

Reply:
AH!
I got you now, silly me...